import math


print(math.sqrt(2))  # this causes an exception that stops the program

print("If you see this, you fixed the error!")
